### PR TITLE
MCP-2498 PDF Template Updates

### PR DESCRIPTION
**Pull request by anthonyvelazquez**
## What was the problem?

- Update the PDF to match the pixels for the table columns from the [Figma document](https://www.figma.com/file/gENy7VJcPxq35DmqCit5av/iMVP-designs?node-id=509%3A1117&t=MDFHQO5OCbeftrJY-0)
- Add display message for the scenario when there are no BP Readings
- Update text for `Page Number(s)` column in Relevant Service Locations

Associated tickets or Slack threads:

- [MCP-2498](https://amida.atlassian.net/browse/MCP-2498)

### CODE REVIEW

File: service-python/pdfgenerator/src/lib/templates/hypertension-v2.html [hunk 1]
```diff
@@ -1,467 +1,489 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta name="pdfkit-orientation" content="Landscape"/>
+    <meta name="pdfkit-footer-right" content="Page [page] of [topage]"/>
+    <meta name="pdfkit-footer-font-name" content="NotoSans"/>
+    <meta name="pdfkit-footer-font-size" content="8"/>
+    <meta name="pdfkit-page-size" content="Legal"/>
+  <!-- 40px -->
+    <meta name="pdfkit-margin-left" content="0.555in"/>
+  <!-- 80px -->
+    <meta name="pdfkit-margin-right" content="1.1in"/>
```
Comment (jbregstone) [inline service-python/pdfgenerator/src/lib/templates/hypertension-v2.html:15]: The page size got larger, I think from this change

File: service-python/pdfgenerator/src/lib/templates/hypertension-v2.html [hunk 2]
```diff
@@ -1,467 +1,489 @@
...
+  <!-- 40px -->
+    <meta name="pdfkit-margin-top" content="0.555in"/>
+  <!-- 20px -->
+    <meta name="pdfkit-margin-bottom" content="0.277in"/>
```
Comment (jbregstone) [inline service-python/pdfgenerator/src/lib/templates/hypertension-v2.html:19]: This margin also changed

File: service-python/pdfgenerator/src/lib/templates/hypertension-v2.html [hunk 3]
```diff
@@ -1,467 +1,489 @@
...
+          <h2 class="m-0 mb-13"><b>Schedular evidence for hypertension</b></h2>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <table id="schedular-evidence" class="table table-bordered">
+            <thead>
+              <tr>
+                <th style="width: 153px !important;">Key terms</th>
+                <th>Observation date</th>
```
Comment (jbregstone) [inline service-python/pdfgenerator/src/lib/templates/hypertension-v2.html:327]: Looks too wide compared to figma document. maybe we should set a standard width for 'Observation date' and 'VBMS receipt date' columns

Comment (jbregstone) [reply service-python/pdfgenerator/src/lib/templates/hypertension-v2.html:327]: Maybe `90 px`

File: service-python/pdfgenerator/src/lib/templates/hypertension-v2.html (hunk 1 above)
Comment (anthonyvelazquez) [reply service-python/pdfgenerator/src/lib/templates/hypertension-v2.html]: Yep, I was trying to fix the DPI issue which required to redo the sizing but looks like that did more harm than good. Redid the PR to only make the main updates

**Review by jbregstone — APPROVED** [review]

**Review by au2 — APPROVED** [review]

**Comment by au2** [comment]
@anthonyvelazquez  please resolve conflicts.

**Pull request merged**